src/constants/apiPaths.ts

```diff
@@ -1,9 +1,8 @@
-
 const API_PATHS = {
-  product: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
-  order: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
-  import: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
-  bff: 'https://.execute-api.eu-west-1.amazonaws.com/dev'
+	product: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
```

Comment:
I assume you should have changed this link and not bff link

Reply:
fixed